winconfig.h

```diff
@@ -93,6 +93,9 @@
 /* Disable advanced SSA JIT optimizations */
 /* #undef DISABLE_SSA */
 
+/* Disable interpreter */
+#define DISABLE_INTERPRETER 1
```

Comment:
why is the interpreter disabled on Windows (the commit message doesn't tell)?

Reply:
it requires changes in the project files for windows and I don't have a windows machine around (it's on my bucket list).